apps/tsilhqotin-language-hub/src/Components/Pages/Videos/Videos.tsx

```diff
@@ -1,14 +1,98 @@
+import { Card } from '@mui/material';
+import { DataGrid, GridColDef, GridRowsProp } from '@mui/x-data-grid';
+import { useEffect, useState } from 'react';
+import { getConfig } from '../../../config';
 import './Videos.module.css';
 
+export type MediaData = {
```

Comment:
Can we call this `MediaViewModel` instead?

Reply:
Renamed to MediaViewModel.